lib/src/controls/form/text_box.dart

```diff
@@ -649,6 +649,7 @@ class _TextBoxState extends State<TextBox>
             : TextEditingValue(text: widget.initialValue!),
       );
     }
+    _effectiveFocusNode.canRequestFocus = enabled;
```

Comment:
this is unecessary since the same is implemented in `didChangeDependencies`

Reply:
Hello, I checked the flutter's TextField widget and it added this line, I'm not sure if this line is redundant